common/djangoapps/student/forms.py

```diff
@@ -115,12 +115,11 @@ class AccountCreationForm(forms.Form):
     validation, not rendering.
     """
     # TODO: Resolve repetition
-    username = forms.SlugField(
+    username = forms.CharField(
```

Comment:
@gwprice looking back in the history it appears you were involved in defining this field a while ago -- just wondering if you have any perspective/context to add.